crates/lang_srv/debug_server.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
```

Comment:
It looks like you have a number of unrelated changes in this file and below. Probably don't want those here?

Reply:
ahh yea this file is not related to this branch. I can remove

the readme/gitignore updates were useful when i was testing the wasm repl so i figured those were fine to leave in but can move out